requirements.txt

```diff
@@ -2,34 +2,51 @@ astroid==2.2.0
 autopep8==1.4.3
 certifi==2018.11.29
 chardet==3.0.4
+coreapi==2.3.3
+coreschema==0.0.4
 coverage==4.5.2
 coveralls==1.6.0
+defusedxml==0.5.0
 dj-database-url==0.5.0
 Django==2.1.7
+django-braces==1.13.0
 django-cors-middleware==1.3.1
 django-extensions==2.1.6
+django-oauth-toolkit==1.2.0
+django-rest-framework-social-oauth2==1.1.0
 djangorestframework==3.9.1
 docopt==0.6.2
+drf-yasg==1.13.0
 entrypoints==0.3
 flake8==3.7.7
 gunicorn==19.9.0
 idna==2.8
+inflection==0.3.1
 isort==4.3.9
+itypes==1.1.0
+Jinja2==2.10
 lazy-object-proxy==1.3.1
+MarkupSafe==1.1.1
 mccabe==0.6.1
+oauthlib==3.0.1
 psycopg2==2.7.7
 psycopg2-binary==2.7.7
 pycodestyle==2.5.0
 pyflakes==2.1.0
 PyJWT==1.4.2
 pylint==2.3.0
 python-decouple==3.1
+python-social-auth==0.3.6
+python3-openid==3.1.0
 pytz==2018.9
 requests==2.21.0
+requests-oauthlib==1.2.0
+ruamel.yaml==0.15.89
 six==1.10.0
+social-auth-app-django==3.1.0
+social-auth-core==3.1.0
 typed-ast==1.3.1
+uritemplate==3.0.0
 urllib3==1.24.1
 whitenoise==4.1.2
 wrapt==1.11.1
-coreapi==2.3.3
```

Comment:
`coreapi` and `drf-yasg` are dependencies for API documentation, would you please restore them.

Reply:
They are there, they just been reordered alphabetically.

Reply:
cool, everything looks good then.